lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -23,12 +27,39 @@ class FirefoxDelegate {
       : 'none';
     this.firefoxConfig = options.firefox || {};
     this.options = options;
+    this.android = null;
     // We keep track of all alias and URLs
     this.aliasAndUrl = {};
+    // Variables for android power testing
+    this.screenBrightnessMode = 0;
+    this.screenBrightness = 127;
   }
 
   async onStart() {
     this.hars = [];
+    if (isAndroidConfigured(this.options)) {
+      this.android = new Android(this.options);
+    }
+
+    if (this.android && this.options.androidPower) {
+      log.info('Initializing power testing');
+
+      // Disable adaptive brightness
+      this.screenBrightnessMode = await this.android._runCommandAndGet(
+        'settings get system screen_brightness_mode'
+      );
+      await this.android._runCommand(
+        'settings put system screen_brightness_mode 0'
+      );
+
+      // Set screen brightness to 50%
+      this.screenBrightness = await this.android._runCommandAndGet(
+        'settings get system screen_brightness'
+      );
+      await this.android._runCommand(
+        'settings put system screen_brightness 127'
```

Comment:
I think we could move all Android commands here as a utility method: https://github.com/sitespeedio/browsertime/blob/master/lib/android/index.js

Then it will be less code to implement it for Chrome and other browser (and easier to find). I think all `.android._runCommand(...)` could be moved to that Android utility file instead.

Reply:
Good point! I'll move all of this start/stop code to that file.

Reply:
Everything is now in the Android class.